### PR TITLE
Work in progress homepage

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

File: components/index/OurTeam.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="c-our-team">
+    <div class="c-our-team__content">
+      <div class="c-our-team__photo">
+        <img src="https://picsum.photos/484/242" />
+      </div>
+      <div class="c-our-team__text">
+        <h1 class="c-our-team__header">Our team &amp; history</h1>
+        <p>We’re a 6 year old foundation with amazing growth and many twists and turns on the way.</p>
```
Comment (heyitsmaimai) [inline components/index/OurTeam.vue]: 7 years old!

File: components/index/Volunteer.vue [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="c-volunteer">
+    <div class="c-volunteer__content">
+      <div class="c-volunteer__illustration">
+        <img src="~/static/images/volunteer.svg" />
+      </div>
+      <div class="c-volunteer__text">
+        <h1>Volunteer at a Dojo</h1>
+        <h2>Join us and meet like 6742 like minded mentors</h2>
+        <p>Whether you can code or not you can help a local club with just a few hours a month! Clubs need general volunteers and mentors. Even mentors don’t have to be technical, they help mainly by encouraging young people to discover their own way forward.</p>
```
Comment (heyitsmaimai) [inline components/index/Volunteer.vue]: Comma after 'Whether you can code or not'

File: components/index/Hero.vue [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      }
+    }
+  }
+
+  @include media-breakpoint-max(sm) {
+    .c-hero {
+      background-image:
+        radial-gradient(ellipse at bottom, $white, $white calc(70% - 2px), transparent 70%, transparent 100%),
+        linear-gradient(to bottom, $cd-purple 0%, $cd-purple-gradient 100%);
+      background-size: 300% 300px, 100%;
```
Comment (heyitsmaimai) [inline components/index/Hero.vue]: tabbing for background- stuff?

Comment (DanielBrierton) [reply components/index/Hero.vue]: The reason this is tabbed in further is that the radial and linear gradients are values for the `background-image`. `background-size` is a new rule then, so falls back in line with the rule names.

**Pull request merged**